pydantic/dataclasses.py

```diff
@@ -233,9 +233,10 @@ def create_dataclass(cls: type[Any]) -> type[PydanticDataclass]:
             if config_wrapper.frozen:
                 # It's not recommended to define both, as the setting from the dataclass decorator will take priority.
                 warn(
-                    f"`frozen` is set via both the 'dataclass' decorator and `config` for dataclass {cls.__name__}."
+                    f'`frozen` is set via both the `dataclass` decorator and `config` for dataclass {cls.__name__}.'
```

Comment:
```suggestion
                    f'`frozen` is set via both the `dataclass` decorator and `config` for dataclass {cls.__name__!r}.'
```
nitpick, this adds single quotes around the name